images/alpine/jdk8/Dockerfile.alpine.jdk8-mvn3.6-mono5-nuget3.4-choco0.10

```
@@ -0,0 +1,22 @@
+FROM mono:3.12.1 as builder
+
+ENV CHOCO_VERSION=master
```

Sorry, I mislead you a bit in our discussions. Could you please switch the version to `9bb4246354188e4205b5dbcae6f8638351eccb13`? We should keep a `fixed revision` until they release `0.10.16`. I don't know if they are planning on introducing breaking changes and it looks like the current `master` revision works for us.

```suggestion
ENV CHOCO_VERSION=9bb4246354188e4205b5dbcae6f8638351eccb13
```

---

Changed.